src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/main.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <main class="page__content">
+      <div class="stars stars--0">
```

Comment:
A better way to use the list for duplicate elements. In this case `<div class="stars__star"></div>` have duplication.

Reply:
Thanks for review, all fixed: https://github.com/mate-academy/layout_stars/pull/4548/commits/712ff9fd53ab59bdcb85a79698ec45111e53b3a9